src/System.Data.SqlClient/src/System/Data/SqlClient/SNI/SNITcpHandle.cs

```diff
@@ -183,13 +183,36 @@ public SNITCPHandle(string serverName, int port, long timerExpire, object callba
 
         private static async Task<Socket> ConnectAsync(string serverName, int port)
         {
-            Socket socket;
-            if (RuntimeInformation.IsOSPlatform(OSPlatform.Windows))
+            // On unix we can only connect using IP addresses, rather than host names, using the instance connect methods.
```

Comment:
Is this comment relevant? We have a unified code path for both Windows and Unix now.